data/styles/clear/include/Roads.mapcss

```diff
@@ -787,6 +794,16 @@ line|z19-[route=hiking],
 line|z19-[highway=footway][sac_scale],
 {width: 4; dashes: 8,4.5;}
 
+line|z16-[highway=path][bridge?]::bridgewhite
+{casing-width-add: 0.5;}
+line|z16-[highway=path][bridge?]::bridgeblack
```

Comment:
for railways bridges we usually display ::bridgewhite only first , e.g. at z13-
and then add ::bridgeblack later at e.g. z16-

I think it works very well and makes bridges visually "ligher weight" at smaller zooms.
Maybe try it with paths/cycleways/etc. also? I.e. add a ::bridgeblack staring at z17- or z18- even.

Reply:
Changed ::bridgewhite to start at z13 and ::bridgeblack at z17

Reply:
::bridgeblack at z17- sounds good!

Bbut ::bridgewhite at z13 is a bit too early, sorry :)

E.g. highway=path is visbible from z14- and highway=footway from z15-, so it doesn't make sense to display ::bridgewhite before the main line.

I suggest make ::bridgewhite start from z15-.

Reply:
👍 That makes sense, changed it to z15